ruff.toml

```diff
@@ -0,0 +1,32 @@
+line-length = 110
```

Comment:
Move this file to pyproject.toml, we have too many config files in root as it is :grimacing: